### ISSUE TITLE
Commits that reference tickets no longer update the tickets they reference

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4368]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#4368|
|Type|release blocker: regression|
|Created|2010-03-12 18:29:35Z|

I just committed r28716 which referenced [[#4346](https://github.com/twisted/twisted/issues/4346)](https://github.com/twisted/twisted/issues/4346) in its commit message. I expected the ticket #4346 to have a new comment made out of my commit message and pointing to that revision, instead, the ticket stayed the same.

<details><summary>Searchable metadata</summary>

```
trac-id__4368 4368
type__release_blocker__regression release blocker: regression
reporter__jml jml
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__website website
keywords__ 
time__1268418575000000 1268418575000000
changetime__1268579873000000 1268579873000000
version__None None
owner__ 
cc__khorn
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn commented|
|-|-|

I also noticed this when committing to [#4239](https://github.com/twisted/twisted/issues/4239)

revision: [28717]

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve commented|
|-|-|

(In [28727]) Retest

Refs [#4368](https://github.com/twisted/twisted/issues/4368)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [28728]) Try closing

Fixes [#4368](https://github.com/twisted/twisted/issues/4368)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
